### DISCUSSION TITLE
what's wrong with this? where can i fix it?

### DISCUSSION THREAD

**Discussion by wangenius** [xyflow/xyflow#4626] [Q&A]
![image](https://github.com/user-attachments/assets/8cc90ea9-c06b-4a8e-96e4-085eecce3498)

**Comment by moklick** [comment]
Could you create a codesandbox https://new.reactflow.dev for this?

**Reply by wangenius** [reply]
it's hard to do that...
![image](https://github.com/user-attachments/assets/95eca37b-b13b-4a59-ac65-6d59c9223c51)
wired

**Reply by wangenius** [reply]
sometimes it's good, but it works like this once reopen
